### PR TITLE
Revise create api

### PR DESCRIPTION
**Pull request by AnitaTT**
revised the create api not to send the message

### CODE REVIEW

**Review by ChiuWeiYi — COMMENTED** [review]
the code looks clear 💯 
curious, how to you test it?

File: slack-announcement-scheduler/db/schema.rb [hunk 1]
```diff
@@ -10,12 +10,32 @@
...
+
   create_table "messages", force: :cascade do |t|
     t.text "name"
     t.text "description"
     t.text "department"
-    t.date "target_announce_date"
+    t.datetime "target_announce_date", precision: nil
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.string "approval"
```
Comment (ChihHaoTW) [inline slack-announcement-scheduler/db/schema.rb:31]: nil -> pending
true -> approved
false -> rejected

Comment (ChihHaoTW) [reply slack-announcement-scheduler/db/schema.rb:31]: can do it in the future

File: slack-announcement-scheduler/db/schema.rb [hunk 2]
```diff
@@ -10,12 +10,32 @@
...
+    t.datetime "target_announce_time"
```
Comment (ChihHaoTW) [inline slack-announcement-scheduler/db/schema.rb]: should delete this

File: slack-announcement-scheduler/app/controllers/api/v1/messages_controller.rb [hunk 3]
```diff
@@ -34,10 +30,28 @@ def update
 
     end
 
+    def accept
+      @message = Message.find(params[:id])
+      @message.status = :accepted
```
Comment (ChihHaoTW) [inline slack-announcement-scheduler/app/controllers/api/v1/messages_controller.rb]:
```suggestion
      @message.approval = :approved
```

File: slack-announcement-scheduler/app/controllers/api/v1/messages_controller.rb [hunk 4]
```diff
@@ -34,10 +30,28 @@ def update
...
+      @message.save
+
+      SendMessageToSlackJob.set(wait_until: Time.parse(@message.target_announce_date.to_s)).perform_later(@message)
+      render json: { message: 'OK' }, status: :ok
+
+    end
+
+    def reject
+      @message = Message.find(params[:id])
+      @message.status = :rejected
```
Comment (ChihHaoTW) [inline slack-announcement-scheduler/app/controllers/api/v1/messages_controller.rb]:
```suggestion
      @message.approval = :rejected
```

File: slack-announcement-scheduler/db/schema.rb (hunk 2 above)
Comment (AnitaTT) [reply slack-announcement-scheduler/db/schema.rb]: only delete "target_announce_time"

**Review by ChihHaoTW — APPROVED** [review]

**Pull request merged**